packages/templates/src/raw/a-day-in-the-life/metaData.js

```diff
@@ -31,9 +31,21 @@ export default {
     _x('Blue', 'template keyword', 'web-stories'),
   ],
   colors: [
-    { label: _x('Waterworld Blue', 'color', 'web-stories'), color: '#061b38' },
-    { label: _x('Arctic White', 'color', 'web-stories'), color: '#fcfcfc' },
-    { label: _x('Honey Pot Yellow', 'color', 'web-stories'), color: '#ffc864' },
+    {
+      label: _x('Waterworld Blue', 'color', 'web-stories'),
+      color: '#061b38',
+      family: _x('Blue', 'color family', 'web-stories'),
+    },
+    {
+      label: _x('Arctic White', 'color', 'web-stories'),
+      color: '#fcfcfc',
+      family: _x('White', 'color family', 'web-stories'),
+    },
+    {
+      label: _x('Honey Pot Yellow', 'color', 'web-stories'),
+      color: '#ffc864',
+      family: _x('Yellow', 'color family', 'web-stories'),
```

Comment:
👋  as resident expert, @swissspidy this usage ok? `_x('Yellow', 'color family', 'web-stories'),`